package/src/components/Channel/Channel.tsx

```diff
@@ -1780,6 +1807,9 @@ const ChannelWithContext = <
     typing,
   });
 
+  // TODO: replace the null view with appropriate message. Currently this is waiting a design decision.
```

Comment:
Added a JIRA task here, since there is not much to discuss. Just need to wait for designs from team - https://stream-io.atlassian.net/browse/CRNS-472?atlOrigin=eyJpIjoiN2Q0Y2JiOTljNTE5NDViY2FlMWJmOWYxZGQxYmM1ZDIiLCJwIjoiaiJ9